### DISCUSSION TITLE
UE5 crashes after about 30 seconds when I open a project.

### DISCUSSION THREAD

**Discussion by NineVoltNed** [JSBSim-Team/jsbsim#1065] [Support requests]
Hello,
I am new to coding and UE5 so bare with me! 
When I open UE5 it loads to main screen and I can open projects and start new ones but as soon as I do I get about 30 seconds and it crashes. I get a full crash report with code lines but when i open in Visual studio i'm not sure what to fix. The crash report is at the bottom. I can include code if need be or open a VS shared file. I have been searching for a fix for this for 3 days now and can't find anything.  
Please help!!

nvwgf2umx
nvwgf2umx
nvwgf2umx
nvwgf2umx
nvwgf2umx
D3D12Core
dxgi
dxgi
dxgi
dxgi
dxgi
dxgi
dxgi
UnrealEditor_D3D12RHI!FD3D12Viewport::PresentChecked() [D:\build\++UE5\Sync\Engine\Source\Runtime\D3D12RHI\Private\D3D12Viewport.cpp:638]
UnrealEditor_D3D12RHI!FD3D12Viewport::Present() [D:\build\++UE5\Sync\Engine\Source\Runtime\D3D12RHI\Private\D3D12Viewport.cpp:709]
UnrealEditor_D3D12RHI!FD3D12CommandContextBase::RHIEndDrawingViewport() [D:\build\++UE5\Sync\Engine\Source\Runtime\D3D12RHI\Private\D3D12Viewport.cpp:916]
UnrealEditor_RHI!FRHICommand&lt;FRHICommandEndDrawingViewport,FRHICommandEndDrawingViewportString2069&gt;::ExecuteAndDestruct() [D:\build\++UE5\Sync\Engine\Source\Runtime\RHI\Public\RHICommandList.h:1245]
UnrealEditor_RHI!FRHICommandListBase::Execute() [D:\build\++UE5\Sync\Engine\Source\Runtime\RHI\Private\RHICommandList.cpp:461]
UnrealEditor_RHI!`FRHICommandListImmediate::ExecuteAndReset&apos;::`35&apos;::&lt;lambda_4&gt;::operator()() [D:\build\++UE5\Sync\Engine\Source\Runtime\RHI\Private\RHICommandList.cpp:772]
UnrealEditor_RHI!TGraphTask&lt;TFunctionGraphTaskImpl&lt;void __cdecl(void),0&gt; &gt;::ExecuteTask() [D:\build\++UE5\Sync\Engine\Source\Runtime\Core\Public\Async\TaskGraphInterfaces.h:1265]
UnrealEditor_Core!FNamedTaskThread::ProcessTasksNamedThread() [D:\build\++UE5\Sync\Engine\Source\Runtime\Core\Private\Async\TaskGraph.cpp:758]
UnrealEditor_Core!FNamedTaskThread::ProcessTasksUntilQuit() [D:\build\++UE5\Sync\Engine\Source\Runtime\Core\Private\Async\TaskGraph.cpp:649]
UnrealEditor_RenderCore!FRHIThread::Run() [D:\build\++UE5\Sync\Engine\Source\Runtime\RenderCore\Private\RenderingThread.cpp:328]
UnrealEditor_Core!FRunnableThreadWin::Run() [D:\build\++UE5\Sync\Engine\Source\Runtime\Core\Private\Windows\WindowsRunnableThread.cpp:149]

**Comment by gallonmate** [comment]
This github repository is for JSBSim related projects. If the crashing of UE5 is happening with non-jsbsim projects, then you should seek help  from UE5 specific communities. That being said: Your crashing appears to be graphics related, specifically some DX12 issue. You can do the following:
-Make sure your graphics card driver is the latest, or if it's already latest, then try downgrading the driver to a previous version.
-Try using low graphic settings in the editor.
-You can switch to DX11 for the editor (you will loose some DX12 only features).
-Click the Verify option for the engine installation found in the Launcher, which will verify the engine files are good.

If you are using UE5.4, then keep in mind it's preview build and could have a bug for your hardware. So try UE5.3
